services/requester/requester.go

```diff
@@ -159,12 +162,36 @@ func (e *EVM) SendRawTransaction(ctx context.Context, data []byte) (common.Hash,
 		return common.Hash{}, err
 	}
 
-	err := models.ValidateTransaction(tx)
-	if err != nil {
+	if err := models.ValidateTransaction(tx); err != nil {
 		return common.Hash{}, err
 	}
 
-	// todo do further validation
+	head := &types.Header{
```

Comment:
This is merely a first functional example, on how to construct the arguments needed for the [ValidateTransaction](https://github.com/ethereum/go-ethereum/blob/master/core/txpool/validation.go#L50-L56) found in `Geth`.
We can choose to move this to our own implementation of `TxPool`, and move there all necessary initialization.